kimchi/src/snarky/poseidon.rs

```diff
@@ -0,0 +1,74 @@
+use super::{
+    checked_runner::Constraint,
+    constraint_system::KimchiConstraint,
+    prelude::{CVar, RunState},
+};
+use crate::loc;
+use ark_ff::PrimeField;
+use itertools::Itertools;
+use oracle::{
+    constants::PlonkSpongeConstantsKimchi, permutation::full_round,
+    poseidon::ArithmeticSpongeParams,
+};
+use std::iter::successors;
+
+const ROUNDS: usize = 55;
+const ROUNDS_PER_ROW: usize = 5;
```

Comment:
I just hardcoded the constants for no particular reasons, any ideas about how to handle them?

Reply:
<img width="554" alt="Screen Shot 2022-10-24 at 10 27 56 AM" src="https://user-images.githubusercontent.com/1316043/197564924-b041b64e-67f1-45e4-8f4c-5130a9bcfd08.png">

polynomials/poseidon.rs should have these already, otherwise maybe a good idea to place them there?

Reply:
I was no sure about using those, I see that there are some other params with for example more rounds, do we have to support those?
I think I will use those from `PlonkSpongeConstantsKimchi` for now and later we can change it or use const generics.